pkg/clustertree/cluster-manager/controllers/pod/root_pod_controller.go

```diff
@@ -689,8 +690,8 @@ func (r *RootPodReconciler) CreatePodInLeafCluster(ctx context.Context, lr *leaf
 		}
 
 		// try to create image pull secrets, ignore err
-		if err = r.createStorageInLeafCluster(ctx, lr, utils.GVR_SECRET, imagePullSecrets, basicPod, clusterNodeInfo); err != nil {
-			klog.Warning(err)
+		if errignore := r.createStorageInLeafCluster(ctx, lr, utils.GVR_SECRET, imagePullSecrets, basicPod, clusterNodeInfo); errignore != nil {
+			klog.Warning(errignore)
 		}
```

Comment:
Is it a bug?

Reply:
> Is it a bug?

yes, it is a bug.  you can see line 698.